pkgs/games/fairymax/default.nix

```diff
@@ -34,7 +34,6 @@ stdenv.mkDerivation rec {
     '';
     license = lib.licenses.free ;
     maintainers = [lib.maintainers.raskin];
-    platforms = lib.platforms.linux;
```

Comment:
Can we set this to platforms.all?

Reply:
Should work, as it's a simple compile on a single file. I pushed the change.